packages/angular/src/utils/mf/utils.ts

```diff
@@ -40,6 +40,9 @@ export async function getModuleFederationConfig(
   determineRemoteUrl: (remote: string) => string,
   options: { isServer: boolean } = { isServer: false }
 ) {
+  //check if mfConfig is from a js or ts configuration and take default values accordingly
+  mfConfig = (mfConfig as any).default ?? mfConfig;
```

Comment:
Is this necessary? Doesn't this PR already return the config from either default export or as is when resolving the config?